CMakeLists.txt

```diff
@@ -108,9 +108,9 @@ install_basic_package_files(${PROJECT_NAME} VARS_PREFIX ${VARS_PREFIX}
 
 include(AddUninstallTarget)
 
-if(ENABLE_TESTS)
+if(YARPWHOLEBODYINTERFACE_ENABLE_TESTS)
 #enable testing
-include(CTest)
+#include(CTest)
```

Comment:
Yeah. My fault. I did't remember the inclusion in other projects,  that's why I commented it.
If it is needed for sure we will add it back.